IGCexpansion/JointAnalysis.py

```diff
@@ -64,10 +67,22 @@ def get_save_file_names(self, save_name):
             model_string = self.Model
 
         if save_name is None:
-            if self.IGC_Omega is None:
-                general_save_name = self.save_path + 'Joint_' + model_string + '_' + str(len(self.paralog_list)) + '_pairs_grand_save.txt'
+            if ReCodonGeneconv.use_IGC_Omega(self):
```

Comment:
I think this will work, since both of the two class function will share the same condition, but I'm not sure. Do you think it will be better and more clear to create a new `use_IGC_Omega/Homo_Omega` function in `JointAnalysis.py`?

Reply:
OK

Reply:
Hello Xiang, the code works. However, now I realize it is unnecessary to use `use_IGC_Omega` to control. Here we can just use `if IGC_Omega/Tau_Omega/Homo_Omega is not None` to control, since all conditions have already been checked when we assign `self.geneconv_list`

Reply:
Sure.  Please make the change.